package.json

```diff
@@ -89,7 +89,8 @@
         }
     },
     "lint-staged": {
-        "*.{js,css,scss}": "prettier --write"
+        "*.{js,css,scss}": "prettier --write",
+        "*.js": "eslint frontend/src"
```

Comment:
Hi, doesn't having `frontend/src` here mean that for every file changed, eslint will check all the files again? That might be a bit too much? Just removing `frontend/src` will make it a bit saner?

Reply:
Yep good shout